tests/test_lca.py

```diff
@@ -1834,3 +1848,330 @@ def test_gather_multiple_return(c):
     print(len(results))
     assert len(results) == 1
     assert results[0][0] == 1.0
+
+
+def test_lca_db_protein_build():
+    # test programmatic creation of LCA database with protein sigs in it
+    sigfile1 = utils.get_test_data('prot/protein/GCA_001593925.1_ASM159392v1_protein.faa.gz.sig')
+    sigfile2 = utils.get_test_data('prot/protein/GCA_001593935.1_ASM159393v1_protein.faa.gz.sig')
+
+    sig1 = sourmash.load_one_signature(sigfile1)
+    sig2 = sourmash.load_one_signature(sigfile2)
+    
+    db = sourmash.lca.LCA_Database(ksize=19, scaled=100, moltype='protein')
```

Comment:
for soumash 3.x, protein kmers should be divisible by 3. Here, 19/3 = 6.3, so the actual ksize being used is k= 6.

We should maybe reimplement the check that #575/ #576 removed?